src/game_character.h

```diff
@@ -1343,7 +1343,6 @@ inline Game_CharacterDataStorage<T>::Game_CharacterDataStorage(Game_CharacterDat
 template <typename T>
 inline Game_CharacterDataStorage<T>& Game_CharacterDataStorage<T>::operator=(Game_CharacterDataStorage&& o) noexcept
 {
-	static_cast<Game_Character*>(this) = std::move(o);
```

Comment:
is that a bugfix?

Reply:
Oh good point. I wanted to revisit that line... should have added a FIXME instead of just deleting it.

That line fails to build with "the left operand must be an l-value" and I couldn't figure out how to solve it (yet).

Reply:
this works. thx